ddtrace/internal/logger.py

```diff
@@ -125,6 +125,13 @@ def handle(self, record):
         :param record: The log record being logged
         :type record: ``logging.LogRecord``
         """
+        if record.levelno >= logging.ERROR:
+            # avoid circular import
+            from ddtrace.internal.telemetry import telemetry_writer
+
+            # currently we only have one error code
+            telemetry_writer.add_error(1, record.msg % record.args)
```

Comment:
can we create an enum somewhere with the error codes, I can see how it'll get really easy to accidentally use the same one in two places


also, what would be the meaning of the error code here?

Reply:
we also need other metadata here for the error, do we get just the message or do we also get the module/logger that was used? (e.g. was this from `ddtrace._monkey` or `ddtrace.internal.writer` ?

(we'd probably be able to tell from the message, but using the error code or including module would make it much easier to query)

Reply:
Ahhh that's a good point. I will add the file name and line number to telemetry error messages